tests/meson.build

```diff
@@ -79,7 +79,8 @@ posix_fail_test_cases = [
 
 glibc_test_cases = [
 	'getopt',
-	'ffsl-ffsll'
+	'ffsl-ffsll',
+	'syscall',
 ]
```

Comment:
This test only works on Linux, it should probably be called `linux-syscall` and only be added if we are compiling for Linux.